### ISSUE TITLE
[Usability] User experiences an unexpected interaction with a component.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80590]
## Value Proposition

As a veteran who is trying to submit a file in the claim status tool, I want to know that the "the files I uploaded..." check box is a required "field" before I try to submit my files, so that I am not surprised to find out I can't submit my file when a validation rule kicks in. 

## Design Notes

![Screenshot 2024-07-01 165337](https://github.com/department-of-veterans-affairs/va.gov-team/assets/10838260/b8fffca1-3827-4b15-ad18-359acadbfb0c)

## Engineering Notes

- Update the AddFilesForm.jsx component so that the va-checkbox has the prop required 

EX: 
![Screenshot 2024-07-02 at 11 14 02 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/4c9d5e50-7e84-4846-8edd-1193d4b892ab)

## Acceptance Criteria

- [ ] On the files tab, there is an indicator the "the files I uploaded..." checkbox is a required field that doesn't require the user to submit the file to see it. 
- [ ] On the document request page, there is an indicator the "the files I uploaded..." checkbox is a required field that doesn't require the user to submit the file to see it. 

"_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User experiences an unexpected interaction with a component.
**VA.gov Experience Standard - category:** Usability
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

On the Files page and the Document request page, the checkbox ""The files I uploaded are supporting documents for this claim only"" is a required field, but there is not an indicator that it is required.

## Recommended action

Indicate that the field is required.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner."

**Comment by pmclaren19** [comment]
Verified in Staging and looks good! @skylerschain @jacobworrell lmk if you have any questions!
